connection.go

```diff
@@ -882,15 +877,8 @@ func (conn *Connection) reader(r io.Reader, c Conn) {
 			}
 			continue
 		} else if code == iproto.IPROTO_CHUNK {
-			if fut = conn.peekFuture(header.RequestId); fut != nil {
-				if err := fut.AppendPush(header, &buf); err != nil {
-					err = ClientError{
-						ErrProtocolError,
-						fmt.Sprintf("failed to append push response: %s", err),
-					}
-					conn.opts.Logger.Report(LogAppendPushFailed, conn, err)
-				}
-			}
+			// IPROTO_CHUNK is unexpected because box.session.push is unsupported.
+			log.Printf("unexpected IPROTO_CHUNK for request %d", header.RequestId)
```

Comment:
`IPROTO_CHUNK` relates to an implementation detail that not everyone understands. It's best to make a statement like this.

A push message is expected, but not supported.

```suggestion
			log.Printf("unsupported box.session.push() for request %d", header.RequestId)
```

Reply:
Please, add a test for the message if possible.